app/src/gui/components/notebook/draft_table.tsx

```diff
@@ -272,18 +252,6 @@ function DraftRecord(props: DraftsRecordProps) {
           }}
         />
       </Box>
-      {not_xs ? (
```

Comment:
why are we removing this toggle?

Reply:
As discussed we're removing the mobile toggle switch and have the responsive layout of the records table